File: algorithms/analysisOfAlgorithms/asymptoticAnalysis/python/set2.py
# Python 3 implementation of the linear search

# Linearly search x in arr[].
# if x is present then return the index,
# otherwise return -1


def search(array, length, element):
    i = 0
    for i in range(i, length):
        if array[i] == element:
            return i
    return -1


# Driver code
arr = [1, 10, 30, 15]
x = 30
n = len(arr)
print(x, " is presented at index ", search(arr, n, x))

File: algorithms/analysisOfAlgorithms/asymptoticAnalysis/javascript/set2.js
function search(arr, length, number) {
    for (let i = 0; i < length; i++) {
        if (arr[i] === number) return i;
    }
    return -1;
}

let arr = [1, 10, 30, 15];
let number = 30;
let length = arr.length;
console.log(`${number} is present at index ${search(arr, length, number)}`);